app/src/main/java/com/duckduckgo/app/onboarding/ui/OnboardingViewModel.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.onboarding.ui
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import com.duckduckgo.app.onboarding.store.OnboardingStore
+
+class OnboardingViewModel(onboardingStore: OnboardingStore) : ViewModel() {
+
+    data class ViewState(
+            val showHome: Boolean
+    )
+
+    val viewState: MutableLiveData<OnboardingViewModel.ViewState> = MutableLiveData()
```

Comment:
cleaner to do this?
`    val viewState = MutableLiveData<OnboardingViewModel.ViewState>()`